src/blocks/file-blocks/explain/line-menu.tsx

```diff
@@ -0,0 +1,28 @@
+import { KebabHorizontalIcon } from "@primer/octicons-react";
+import { ActionList, ActionMenu, IconButton } from "@primer/react";
+
+export function LineMenu(props: { onExplain: () => void }) {
+  const { onExplain } = props;
+  return (
+    <ActionMenu>
+      <ActionMenu.Anchor>
+        <IconButton
+          size="small"
+          sx={{ height: 25 }}
+          icon={KebabHorizontalIcon}
+          aria-label="Open column options"
+        />
+      </ActionMenu.Anchor>
+
+      <ActionMenu.Overlay>
+        <ActionList>
+          <ActionList.Item disabled>Copy Line</ActionList.Item>
+          <ActionList.Item disabled>Copy Permalink</ActionList.Item>
+          <ActionList.Item disabled>View git blame</ActionList.Item>
```

Comment:
I'm guessing you have these in there to make it feel like the github.com ui? it might be a bit confusing having them there, but always disabled. maybe worth hooking these up? not necessary, just a thought.

Reply:
Will do in a follow-up PR!